primitives/staking/src/offence.rs

```diff
@@ -26,8 +26,6 @@ use crate::SessionIndex;
 
 /// The kind of an offence, is a byte string representing some kind identifier
 /// e.g. `b"im-online:offlin"`, `b"babe:equivocatio"`
-// TODO [slashing]: Is there something better we can have here that is more natural but still
```

Comment:
Is this TODO addressed by this PR?

Reply:
This TODO should never end up being committed.